app/.env.example

```diff
@@ -5,12 +5,13 @@ VUE_APP_REGISTRY_HOME_URL="https://dev.bcregistry.gov.bc.ca/"
 VUE_APP_AUTH_WEB_URL="https://dev.account.bcregistry.gov.bc.ca/"
 VUE_APP_BUSINESSES_URL="https://dev.account.bcregistry.gov.bc.ca/"
 VUE_APP_CORPORATE_ONLINE_URL="https://www.corporateonline.gov.bc.ca"
-VUE_APP_DASHBOARD_URL="https://dev.business.bcregistry.gov.bc.ca/"
+VUE_APP_BUSINESS_DASH_URL="https://dev.business.bcregistry.gov.bc.ca/"
 VUE_APP_ENTITY_SELECTOR_URL="https://entity-selection-dev.apps.silver.devops.gov.bc.ca/"
 VUE_APP_PAYMENT_PORTAL_URL="https://dev.account.bcregistry.gov.bc.ca/makepayment/"
 VUE_APP_SITEMINDER_LOGOUT_URL="https://logontest7.gov.bc.ca/clp-cgi/logoff.cgi"
 VUE_APP_SOCIETIES_ONLINE_HOME_URL="https://dev.bcregistry.ca/societies/"
 VUE_APP_STEPS_TO_RESTORE_URL="https://www2.gov.bc.ca/gov/content/employment-business/business/managing-a-business/permits-licences/businesses-incorporated-companies/incorporated-companies#restore"
+VUE_APP_BUSINESS_REGISTRY_URL="https://dev.business-registry-dashboard.bcregistry.gov.bc.ca/"
```

Comment:
Why don't you sort the keys in this section (and also in vaults file) to make them easier to find visually?